components/BinanceImage/index.tsx

```diff
@@ -0,0 +1,17 @@
+import BinanceImage from 'public/icons/binance-image'
+import React from 'react'
+
+const BinanceIcon = () => {
+
+    return (
+        <>
+            <div className='grid md:grid-cols-2 md:space-y-0 space-y-12 sticky'>
+                <div className='col-span-1'>
+                    <BinanceImage />
+                </div>
+            </div>
+        </>
+    )
+}
+
+export default BinanceIcon
```

Comment:
Please this isn't an icon, but an image. The component name is confusing

Reply:
Also, please is there a reason this is a separate component in the components folder? It's just an image that's used probably one time.

Reply:
@MeisterJustice  I can change the name of the component to something better secondl  I made it a component because there are other images that are used in different files..

I felt this would make it neater and more organized incase someone else wants to work on the code base.

I followed the guide someone else used in the previous pr that was merged.. although he imported his image inside another component

Reply:
Please abstract that out. The components folder should either be for reusable components or page components. Imagine we start creating all images as components in the component folder. That folder would be difficult to navigate

Reply:
Okay noted, how do you want the images rendered.. 
Any specific way

Reply:
The component is just 4 lines of code, so it can go into content/docs/payments/test-payments.mdx